testharness/src/main/java/com/google/accompanist/testharness/ContextThemeWrapper.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.testharness;
+
+import android.content.Context;
+import android.content.ContextWrapper;
+import android.content.res.AssetManager;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.os.Build;
+import android.view.LayoutInflater;
+
+import androidx.annotation.RequiresApi;
+import androidx.annotation.StyleRes;
+
+/**
+ * A context wrapper that allows you to modify or replace the theme of the wrapped context.
+ * <p>
+ * This is a modification of androidx.appcompat.view.ContextThemeWrapper to allow the
+ * TestHarness to work in previews.
+ */
+class ContextThemeWrapper extends ContextWrapper {
```

Comment:
I kept this class in `.java`, to be closer to the appcompat `ContextThemeWrapper`. I could convert and simplify, if desired.

Reply:
If you are happy to maintain Java code, fine by me!